Perpetuum.DataDumper/Factory/DataDumper.ModuleRemoteArmorRepair.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Perpetuum.DataDumper.Views;
+using Perpetuum.ExportedTypes;
+using Perpetuum.Items.Ammos;
+using Perpetuum.Modules;
+using Perpetuum.Modules.Weapons;
+using Perpetuum.Services.ExtensionService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Perpetuum.DataDumper.DataDumper;
+
+namespace Perpetuum.DataDumper { 
+    public partial class DataDumper {
+        public ModuleRemoteArmorRepairerDataView NewModuleRemoteArmorRepairerDataView(ActiveModule module) {
+            var newView = new ModuleRemoteArmorRepairerDataView();
+            InitActiveModuleView(newView, module);
+
+            newView.module_optimal_range = module.GetBasePropertyModifier(AggregateField.optimal_range).Value * 10;
+            newView.module_repair_amount = module.GetBasePropertyModifier(AggregateField.armor_repair_amount).Value;
+            // module.GetBasePropertyModifier(AggregateField.heal)
```

Comment:
?

Reply:
I guess the question is about the * 10? For some reason all of the "range" are stored in meters / 10. This could be refactored to a function like `RangePropertyToMeters(double value)` that just multiplies by 10 but makes it more clear what's going on.

Reply:
oh no that's fine, just the commented out heal is what I was referring to.  I did this review backwards apparently (see comment below on the same comment line)